addons/web/static/tests/legacy/fields/relational_fields_tests.js

```diff
@@ -787,7 +787,7 @@ QUnit.module('Legacy relational_fields', {
         });
 
         assert.strictEqual(count, 1, 'once search_read should have been done to fetch the relational values');
-        assert.strictEqual(nb_fields_fetched, 1, 'search_read should only fetch field id');
+        assert.strictEqual(nb_fields_fetched, 2, 'search_read should only fetch field id and display_name');
```

Comment:
I would do a deepEqual instead (and maybe remove "id" from the list anyway as it is useless)